src/group_theory/exponent.lean

```diff
@@ -26,22 +27,28 @@ the lowest common multiple of the order of all elements of the group `G`.
 
 ## Main results
 
-* `lcm_order_eq_exponent`: For a finite group `G`, the exponent is equal to the `lcm` of the order
-  of its elements.
+* `monoid.lcm_order_eq_exponent`: For a finite left cancel monoid `G`, the exponent is equal to the
+  `finset.lcm` of the order of its elements.
+* `is_cyclic.exponent_eq_card`: For a finite cyclic group `G`, the exponent is equal to
+  the group's cardinality.
+* `is_cyclic.exponent_eq_zero_of_infinite`: Infinite cyclic groups have exponent zero.
```

Comment:
I would find it more natural if these results were moved to `group_theory.specific_groups.cyclic`. Exponent should only develop the general theory of exponents, if it is specifically about cyclic groups, it should be proved in that file.

Reply:
the namespacing should've been a hint to me!